processed 4 tasks

task 1 'publish'. lines 4-26:
Error: VMError with status STLOC_UNSAFE_TO_DESTROY_ERROR at location Module ModuleId { address: f75daa73fc071f93593335eb9033da804777eb94491650dd3f095ce6f778acb6, name: Identifier("m") } and message Backtrace: Backtrace [
    { fn: "std::backtrace_rs::backtrace::libunwind::trace", file: "/rustc/9b00956e56009bab2aa15d7bff10916599e3d6d6/library/std/src/../../backtrace/src/backtrace/libunwind.rs", line: 105 },
    { fn: "std::backtrace_rs::backtrace::trace_unsynchronized", file: "/rustc/9b00956e56009bab2aa15d7bff10916599e3d6d6/library/std/src/../../backtrace/src/backtrace/mod.rs", line: 66 },
    { fn: "std::backtrace::Backtrace::create", file: "/rustc/9b00956e56009bab2aa15d7bff10916599e3d6d6/library/std/src/backtrace.rs", line: 331 },
    { fn: "move_binary_format::errors::PartialVMError::finish", file: "/Users/lightmark/aptos-core/third_party/move/move-binary-format/src/errors.rs", line: 389 },
    { fn: "move_bytecode_verifier::code_unit_verifier::CodeUnitVerifier::verify_module::{{closure}}", file: "/Users/lightmark/aptos-core/third_party/move/move-bytecode-verifier/src/code_unit_verifier.rs", line: 43 },
    { fn: "core::result::Result<T,E>::map_err", file: "/rustc/9b00956e56009bab2aa15d7bff10916599e3d6d6/library/core/src/result.rs", line: 829 },
    { fn: "move_bytecode_verifier::code_unit_verifier::CodeUnitVerifier::verify_module", file: "/Users/lightmark/aptos-core/third_party/move/move-bytecode-verifier/src/code_unit_verifier.rs", line: 42 },
    { fn: "move_bytecode_verifier::verifier::verify_module_with_config::{{closure}}", file: "/Users/lightmark/aptos-core/third_party/move/move-bytecode-verifier/src/verifier.rs", line: 121 },
    { fn: "std::panicking::try::do_call", file: "/rustc/9b00956e56009bab2aa15d7bff10916599e3d6d6/library/std/src/panicking.rs", line: 552 },
    { fn: "___rust_try" },
    { fn: "std::panicking::try", file: "/rustc/9b00956e56009bab2aa15d7bff10916599e3d6d6/library/std/src/panicking.rs", line: 516 },
    { fn: "std::panic::catch_unwind", file: "/rustc/9b00956e56009bab2aa15d7bff10916599e3d6d6/library/std/src/panic.rs", line: 146 },
    { fn: "move_bytecode_verifier::verifier::verify_module_with_config", file: "/Users/lightmark/aptos-core/third_party/move/move-bytecode-verifier/src/verifier.rs", line: 95 },
    { fn: "move_bytecode_verifier::verifier::verify_module", file: "/Users/lightmark/aptos-core/third_party/move/move-bytecode-verifier/src/verifier.rs", line: 59 },
    { fn: "<aptos_transactional_test_harness::aptos_test_harness::AptosTestAdapter as move_transactional_test_runner::framework::MoveTestAdapter>::publish_module", file: "./src/aptos_test_harness.rs", line: 844 },
    { fn: "move_transactional_test_runner::framework::MoveTestAdapter::handle_command", file: "/Users/lightmark/aptos-core/third_party/move/testing-infra/transactional-test-runner/src/framework.rs", line: 459 },
    { fn: "move_transactional_test_runner::framework::handle_known_task", file: "/Users/lightmark/aptos-core/third_party/move/testing-infra/transactional-test-runner/src/framework.rs", line: 1115 },
    { fn: "move_transactional_test_runner::framework::run_test_impl", file: "/Users/lightmark/aptos-core/third_party/move/testing-infra/transactional-test-runner/src/framework.rs", line: 1053 },
    { fn: "aptos_transactional_test_harness::aptos_test_harness::run_aptos_test_with_config", file: "./src/aptos_test_harness.rs", line: 1166 },
    { fn: "tests::runner", file: "./tests/tests.rs", line: 21 },
    { fn: "datatest_stable::runner::Requirements::expand::{{closure}}::{{closure}}", file: "/Users/lightmark/.cargo/registry/src/index.crates.io-6f17d22bba15001f/datatest-stable-0.1.3/src/runner.rs", line: 59 },
    { fn: "libtest_mimic::Trial::test::{{closure}}", file: "/Users/lightmark/.cargo/registry/src/index.crates.io-6f17d22bba15001f/libtest-mimic-0.5.2/src/lib.rs", line: 110 },
    { fn: "core::ops::function::FnOnce::call_once{{vtable.shim}}", file: "/rustc/9b00956e56009bab2aa15d7bff10916599e3d6d6/library/core/src/ops/function.rs", line: 250 },
    { fn: "<alloc::boxed::Box<F,A> as core::ops::function::FnOnce<Args>>::call_once", file: "/rustc/9b00956e56009bab2aa15d7bff10916599e3d6d6/library/alloc/src/boxed.rs", line: 2020 },
    { fn: "libtest_mimic::run_single::{{closure}}", file: "/Users/lightmark/.cargo/registry/src/index.crates.io-6f17d22bba15001f/libtest-mimic-0.5.2/src/lib.rs", line: 500 },
    { fn: "<core::panic::unwind_safe::AssertUnwindSafe<F> as core::ops::function::FnOnce<()>>::call_once", file: "/rustc/9b00956e56009bab2aa15d7bff10916599e3d6d6/library/core/src/panic/unwind_safe.rs", line: 272 },
    { fn: "std::panicking::try::do_call", file: "/rustc/9b00956e56009bab2aa15d7bff10916599e3d6d6/library/std/src/panicking.rs", line: 552 },
    { fn: "___rust_try" },
    { fn: "std::panicking::try", file: "/rustc/9b00956e56009bab2aa15d7bff10916599e3d6d6/library/std/src/panicking.rs", line: 516 },
    { fn: "std::panic::catch_unwind", file: "/rustc/9b00956e56009bab2aa15d7bff10916599e3d6d6/library/std/src/panic.rs", line: 146 },
    { fn: "libtest_mimic::run_single", file: "/Users/lightmark/.cargo/registry/src/index.crates.io-6f17d22bba15001f/libtest-mimic-0.5.2/src/lib.rs", line: 500 },
    { fn: "libtest_mimic::run::{{closure}}", file: "/Users/lightmark/.cargo/registry/src/index.crates.io-6f17d22bba15001f/libtest-mimic-0.5.2/src/lib.rs", line: 471 },
    { fn: "<F as threadpool::FnBox>::call_box", file: "/Users/lightmark/.cargo/registry/src/index.crates.io-6f17d22bba15001f/threadpool-1.8.1/src/lib.rs", line: 95 },
    { fn: "threadpool::spawn_in_pool::{{closure}}", file: "/Users/lightmark/.cargo/registry/src/index.crates.io-6f17d22bba15001f/threadpool-1.8.1/src/lib.rs", line: 769 },
    { fn: "std::sys_common::backtrace::__rust_begin_short_backtrace", file: "/rustc/9b00956e56009bab2aa15d7bff10916599e3d6d6/library/std/src/sys_common/backtrace.rs", line: 155 },
    { fn: "std::thread::Builder::spawn_unchecked_::{{closure}}::{{closure}}", file: "/rustc/9b00956e56009bab2aa15d7bff10916599e3d6d6/library/std/src/thread/mod.rs", line: 528 },
    { fn: "<core::panic::unwind_safe::AssertUnwindSafe<F> as core::ops::function::FnOnce<()>>::call_once", file: "/rustc/9b00956e56009bab2aa15d7bff10916599e3d6d6/library/core/src/panic/unwind_safe.rs", line: 272 },
    { fn: "std::panicking::try::do_call", file: "/rustc/9b00956e56009bab2aa15d7bff10916599e3d6d6/library/std/src/panicking.rs", line: 552 },
    { fn: "___rust_try" },
    { fn: "std::panicking::try", file: "/rustc/9b00956e56009bab2aa15d7bff10916599e3d6d6/library/std/src/panicking.rs", line: 516 },
    { fn: "std::panic::catch_unwind", file: "/rustc/9b00956e56009bab2aa15d7bff10916599e3d6d6/library/std/src/panic.rs", line: 146 },
    { fn: "std::thread::Builder::spawn_unchecked_::{{closure}}", file: "/rustc/9b00956e56009bab2aa15d7bff10916599e3d6d6/library/std/src/thread/mod.rs", line: 527 },
    { fn: "core::ops::function::FnOnce::call_once{{vtable.shim}}", file: "/rustc/9b00956e56009bab2aa15d7bff10916599e3d6d6/library/core/src/ops/function.rs", line: 250 },
    { fn: "<alloc::boxed::Box<F,A> as core::ops::function::FnOnce<Args>>::call_once", file: "/rustc/9b00956e56009bab2aa15d7bff10916599e3d6d6/library/alloc/src/boxed.rs", line: 2020 },
    { fn: "<alloc::boxed::Box<F,A> as core::ops::function::FnOnce<Args>>::call_once", file: "/rustc/9b00956e56009bab2aa15d7bff10916599e3d6d6/library/alloc/src/boxed.rs", line: 2020 },
    { fn: "std::sys::pal::unix::thread::Thread::new::thread_start", file: "/rustc/9b00956e56009bab2aa15d7bff10916599e3d6d6/library/std/src/sys/pal/unix/thread.rs", line: 108 },
    { fn: "__pthread_joiner_wake" },
] at index 0 for function definition at code offset 11 in function definition 0

task 2 'run'. lines 28-28:
Error: Failed to execute transaction. ExecutionStatus: MiscellaneousError(None)

task 3 'run'. lines 30-44:
Error: Failed to execute transaction. ExecutionStatus: MiscellaneousError(None)
